packages/hoppscotch-backend/src/admin/admin.service.ts

```diff
@@ -240,14 +240,26 @@ export class AdminService {
       teamID,
       user.value.uid,
     )();
-    if (E.left(isUserAlreadyMember)) {
+    if (E.isLeft(isUserAlreadyMember)) {
       const addedUser = await this.teamService.addMemberToTeamWithEmail(
         teamID,
         userEmail,
         role,
       );
       if (E.isLeft(addedUser)) return E.left(addedUser.left);
 
+      const isUserInvited =
```

Comment:
```suggestion
      const teamInvitation =
```
As it is not a boolean type value, we can change the variable name.

Reply:
cool